android/app/src/main/java/org/cryptimeleon/incentive/app/data/CryptoRepository.kt

```diff
@@ -69,8 +69,7 @@ class CryptoRepository(
             incentiveSystem.generateJoinRequest(providerPublicKey, userKeyPair, promotionParameters)
         val joinResponse = cryptoApiService.runIssueJoin(
             jsonConverter.serialize(joinRequest.representation),
-            promotionParameters.promotionId.toString(),
-            jsonConverter.serialize(userKeyPair.pk.representation)
+            promotionParameters.promotionId.toString()
```

Comment:
why werent the promotion parameters needed until now?

Reply:
They were :thinking:
I only removed the upk